templates/registration/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <div>
+    <h5>Login in your account</h5>
+    {% if form.errors %}
+      <p style="color: red">{{ form.errors }}Login or password is wrong</p>
```

Comment:
The error message concatenates `form.errors` with a static message, which might not be clear. Consider displaying `form.errors` separately or providing a more user-friendly message to improve clarity.